src/utils.ts

```diff
@@ -1860,15 +1860,47 @@ export class newText extends newShape {
   }
 }
 
+export interface pointStyle {
+  size?: number, fillStyle?: string, strokeStyle?: string, marker?: string, markerOrientation?: string
+}
+
+export class newPointStyle implements pointStyle {
+  public size: number;
+  public fillStyle: string;
+  public strokeStyle: string;
+  public marker: string;
+  public markerOrientation: string;
+  constructor(
+    { size = null,
```

Comment:
I am bugged out by this. I find this elegant in a way as well as absolutely horrendous. What happens if you :

```ts
constructor(
  {
     public size = null,
     public fillStyle = null,
     ...
   }
)
```

OR 

```ts
constructor(
  public {
     size = null,
     fillStyle = null,
     ...
   }
)
```

Normally, if you set your args with public in your constructor, you don't need to set them again before. For ex : 

```ts
export class X implements Y {
  constructor(
    public size,
    public fillStyle,
    ...
  )
}
```

Reply:
This way I can build a PointStyle with a nearly perfect equivalent to kwargs of Python.
![image](https://github.com/Dessia-tech/plot_data/assets/106539650/f7902342-51f7-41a7-8bec-fc9dbe1d9fb2)

https://stackoverflow.com/questions/61154670/how-to-imitate-kwargs-from-python-in-typescript

Reply:
This allows to create a pointstyle like it `new PointStyle({strokeStyle: "hsl"0,0%,100%)"})` for example, where strokeStyle is the third argument.